samples/detect.v2beta1.js

```diff
@@ -0,0 +1,822 @@
+    projectId: projectId,
+  });
+
+  /**
+   * TODO(developer): Uncomment the following lines before running the sample.
+   */
+  //const projectId = `your GCLOUD project name, if not specified as env variable e.g. my-Gcloud-Project`;
+  //const knowledgeBaseFullName = `the full path of your knowledge base, e.g my-Gcloud-project/myKnowledgeBase`;
+
+  client
+    .listDocuments({parent: knowledgeBaseFullName})
+    .then(responses => {
+      var resources = responses[0];
+      console.log(
+        `There are ${resources.length} documents in ${knowledgeBaseFullName}`
+      );
+      resources.forEach(r => {
+        console.log(` KnowledgeType: ${r.knowledgeType}`);
+        console.log(` displayName: ${r.displayName}`);
+        console.log(` mimeType: ${r.mimeType}`);
+        console.log(` contentUri: ${r.contentUri}`);
+        console.log(` source: ${r.source}`);
+        console.log(` name: ${r.name}`);
+      });
+    })
+    .catch(err => {
+      console.error(err);
+    });
+  // [END dialogflow_list_document]
+}
+
+function getDocument(documentId) {
+  // [START dialogflow_get_document]
+  // Imports the Dialogflow client library
+  const dialogflow = require('dialogflow').v2beta1;
+
+  // Instantiate a DialogFlow Documents client.
+  var client = new dialogflow.DocumentsClient({
+    projectId: projectId,
+  });
+
+  /**
+   * TODO(developer): Uncomment the following lines before running the sample.
+   */
+  //const documentId = `full path to document in knowledge base, e.g. myKnowledgeBase/documents/myDoc`;
+
+  client
+    .getDocument({name: documentId})
+    .then(responses => {
+      var r = responses[0];
+      console.log(` KnowledgeType: ${r.knowledgeType}`);
+      console.log(` displayName: ${r.displayName}`);
+      console.log(` mimeType: ${r.mimeType}`);
+      console.log(` contentUri: ${r.contentUri}`);
+      console.log(` source: ${r.source}`);
+      console.log(` name: ${r.name}`);
+    })
+    .catch(err => {
+      console.error(err);
+    });
+  // [END dialogflow_get_document]
+}
+
+function deleteDocument(projectId, documentId) {
+  // [START dialogflow_delete_document]
+  // Imports the Dialogflow client library
+  const dialogflow = require('dialogflow').v2beta1;
+
+  // Instantiate a DialogFlow Documents client.
+  var client = new dialogflow.DocumentsClient({
+    projectId: projectId,
+  });
+
+  /**
+   * TODO(developer): Uncomment the following lines before running the sample.
+   */
+  //const projectId = `your GCLOUD project name, if not specified as env variable e.g. my-Gcloud-Project`;
+  //const documentId = `full path to document in knowledge base, e.g. myKnowledgeBase/documents/myDoc`;
+
+  client
+    .deleteDocument({name: documentId})
+    .then(console.log(`document deleted`))
+    .catch(err => {
+      console.error(err);
+    });
+  // [END dialogflow_delete_document]
+}
+
+//v2b1 features
+function detectIntentandSentiment(projectId, sessionId, query, languageCode) {
```

Comment:
nit: How about `detectIntentWithSentiment` – `Intentand` the `And` should be camel**Case**